### DISCUSSION TITLE
Release Cycle

### DISCUSSION THREAD

**Discussion by maddes8cht** [microsoft/terminal#15915] [General]
It was said that Windows terminal "changed to a quarterly realese cycle".
It is not apparent to me what the previous release cycle was - presumably none - but at present I find it unsatisfactory to have no indication of approximately when a new release might be expected.
In VSCode you can see that at some point the endgame is first announced, then goes through its stages, and then it is clear within what timeframe a release is expected.
Here you see - nothing.
The last release was on May 23 (the quick bug fix from May 29 does not count as a release). It's okay if it takes a few days longer, but here it is so that I simply do not see any clue whether anything at all happens in the direction of release - or if i have to expect that in a few months i will see a message like: 
"our new quarterly release cycle didn't live much longer than a single release" ... 
I would love to see some more transparency here, like it is in vscode.

By the way:
I really love windows terminal, so thanks for the great work !

**Comment by maddes8cht** [comment]
There was once a [Roadmap](https://github.com/microsoft/terminal/blob/main/doc/terminal-v2-roadmap.md) that was replaced in 2022 by a new [Roadmap2022](https://github.com/microsoft/terminal/blob/main/doc/roadmap-2022.md). 
Both have now been completely outdated for more than a year.

**Comment by zadjii-msft** [comment]
Yea that's my bad - I'm idly working on a fresh roadmap (tracked in #15661) while waiting for code to compile. It's definitely in progress, just... slowly. Hopefully I'll post that later this week. Thanks for bearing with us!

**Comment by maddes8cht** [comment]
Ah, thanks for the reply.
it doesn't sound like a release will be happening in the near future though ...

**Reply by zadjii-msft** [reply]
Oh, no we're planning on that for later in September ☺️
